### ISSUE TITLE
FSE: Preview v. Live Site View

### ISSUE THREAD

**Issue by retnonindya** [Automattic/wp-calypso#67358]
### Quick summary

When the user changed the template, the "default" template is still visible on Preview, while the live site respects the changes on the editor.

This bug report came from a user. They mentioned that they set the page title ("Exipure Brown Cells"), like this:

![Screenshot 2022-09-04 at 11 28 25 AM](https://user-images.githubusercontent.com/44726232/188295837-1f16f05c-3b94-48b3-b407-f6d2479d9703.png)

They have removed the Page Title Block from Site Editor, but when they Previewed it, they still see the page title:

![Screenshot 2022-09-04 at 11 30 12 AM](https://user-images.githubusercontent.com/44726232/188295866-7a8874b2-c1ba-460a-94ea-2db1be235d93.png)

Although on the live site, the page title is not visible:

![Screenshot 2022-09-04 at 11 31 52 AM](https://user-images.githubusercontent.com/44726232/188295910-449cfafb-241a-4361-bc49-cc5d781602eb.png)

A bit of note, though, the page title is visible when they visited the site when logged in.

This caused confusion, and the user would like to have both Preview and Live Site display the same thing (the final version.)

### Steps to reproduce

1. Start at Appearance > Editor
2. Do changes to the template. On this case, template "Page"
3. Remove the Page Title Block so the title won't be visible on the live site (but site admins/editors can still add page title to make it easier for them to manage the site)
4. Go back to page editor. Choose any page.
5. Add the title on the page with the template "Page" on it
6. Click "Preview". The page title still visible on Preview
7. Check the live site. The page title is not visible.

### What you expected to happen

For Preview and live site to display the expected results (no Page Title)

### What actually happened

Preview still shows page title, while live site does not

### Context

Customer report: 36762875-hc

### Platform (Simple, Atomic, or both?)

Atomic

### Reproducibility

Once

### Severity

One

### Available workarounds?

No but the platform is still usable

### Workaround details

No workaround. I can only inform the user that the live site display is the one that matters.

**Comment by Greatdane** [comment]
📌 SCRUBBING : RESULT
- Tested on Simple - Could Not Replicate
- Tested on AT - Could Not Replicate

📌 FINDINGS/SCREENSHOTS/VIDEO
Cannot reproduce, at step 6, '_Click "Preview". The page title still visible on Preview_', when I preview a page with the template that has the title block removed, the preview does not show the title. The page editor still does of course, but the preview works as expected and shows the same as the live site. 

VIdeo;


https://user-images.githubusercontent.com/5075840/189035768-94c8afe9-8637-4a49-83f2-3e7d51136699.mp4



📌 ACTIONS
Await author feedback

📌 Message to Author
@retnonindya I am struggling to reproduce this. Some questions on your steps;

> Go back to page editor. Choose any page.
Do you mean edit the page using My Site → Pages → All Pages and choosing a page to edit?

If so, I did that and when previewing the title does not show up.

**Comment by retnonindya** [comment]
Thanks for checking this, @Greatdane!

> Do you mean edit the page using My Site → Pages → All Pages and choosing a page to edit?

That would be it, yep.

I rechecked the user's site, and it seems like they decided to put up the page title after all, so at this point it seems like we can't do anything :/ I'm so sorry for the hassle. Since it can't be replicated (I also tried, and I can't replicate it on my end,) it might be plugin/theme conflict. 

I'm going to close the GH issue here. Thanks again, David, for looking into this! 🙇
